src/fetch_family_allowance.py

```diff
@@ -0,0 +1,91 @@
+import datetime
+import warnings
+import requests
+import zipfile
+from urllib.request import urlretrieve
+import os
+import pandas
+import fileinput
+from sqlalchemy import create_engine
+import sys
+import shutil
+
+STARTING_YEAR = 2011
+
+NOW = datetime.datetime.now()
+
+SERVER = 'http://arquivos.portaldatransparencia.gov.br/'
+PATH = 'downloads.asp?a={}&m={:02d}&consulta=BolsaFamiliaFolhaPagamento'
+outputfile = 'data_family_allowance.csv'
+BASE_DATA_DIR = '../data/'
+TEMP_DATA = "../data/temp_data/"
+
+columns = {   'UF': 'FEDERAL_UNIT',
+              'CódigoSIAFIMunicípio': 'CODE_SIAFI_TONW',
+              'NomeMunicípio': 'TOWN_NAME',
+              'CódigoFunção': 'FUNCTION_CODE',
+              'CódigoSubfunção': 'SUBFUNCTION_CODE',
+              'CódigoPrograma': 'PROGRAM_CODE',
+              'CódigoAção': 'ACTION_CODE',
+              'NISFavorecido': 'FAVORED_NIS',
+              'NomeFavorecido': 'FAVORED_NAME',
+              'Fonte-Finalidade': 'SOURCE_FINALITY',
+              'ValorParcela': 'PARCEL_VALUE',
+              'MêsCompetência': 'MONTH_COMPETENCE',
+          }
+
+def dlProgress(count, blockSize, totalSize):
+  percent = int(count*blockSize*100/totalSize)
+  sys.stdout.write("\r" + "progress" + "...%d%%" % percent)
+  sys.stdout.flush()
```

Comment:
We can replace these two lines with `print(…, end='/r')` — it's simpler and cleaner.